src/odoc/occurrences.ml

```diff
@@ -0,0 +1,205 @@
+open Or_error
+
+let handle_file file ~f =
+  Odoc_file.load file
+  |> Result.map @@ fun unit' ->
```

Comment:
That's not compatible.

Reply:
CI i now passing, compatibility should be achieved...